File: Javascript/8-Kyu/Calculate-BMI.js
// Calculate BMI
function bmi(weight, height) {
    
    let bmi = (weight / (height ** 2)).toFixed(1);
    let ranges = [18.5, 25, 30, 100];
    let classes = ['Underweight', 'Normal', 'Overweight', 'Obese'];
    let output = '';

    for (idx = (ranges.length - 1); bmi <= ranges[idx]; idx--) {
        output = classes[idx];

    };

    return output;

};

// Tests
console.log(bmi(60, 1.80)); // --> "Underweight"
console.log(bmi(80, 1.80)); // --> "Normal"
console.log(bmi(90, 1.80)); // --> "Overweight"
console.log(bmi(100, 1.80)); // --> "Obese"

File: Python/8-Kyu/Calculate-BMI.py
# Calculate BMI
def bmi(weight, height):

    calc = weight / (height ** 2)

    return 'Underweight' if calc <= 18.5 else 'Normal' if calc <= 25.0 else 'Overweight' if calc <= 30.0 else 'Obese'

# Tests
print(bmi(50, 1.80))  # --> "Underweight"
print(bmi(80, 1.80))  # --> "Normal"
print(bmi(90, 1.80))  # --> "Overweight"
print(bmi(110, 1.80))  # --> "Obese"
print(bmi(50, 1.50))  # --> "Normal"
